grug/testing/tests/cron.rs

```diff
@@ -36,6 +41,18 @@ mod tester {
 
         Ok(Response::new().add_message(Message::transfer(job.receiver, job.coin).unwrap()))
     }
+
+    pub fn failing_instantiate(_: MutableCtx, _: Empty) -> StdResult<Response> {
+        Ok(Response::new())
+    }
+
+    pub fn failing_cron_execute(ctx: SudoCtx) -> StdResult<Response> {
+        SHOULD_BE_EMPTY.save(ctx.storage, &()).unwrap();
```

Comment:
instead of creating a new `Item`, it's easier to just do `ctx.storage.set(b"foo", b"bar")`